tests/test_sort_package_xml/package.ng.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>rospkg</name>
+  <version>0.0.0</version>
+  <description>The dummy package to test pre-commit hooks</description>
+  <maintainer email="isamu.takagi@tier4.jp">Takagi, Isamu</maintainer>
+  <license>Apache License 2.0</license>
+
+  <buildtool_depend>ament_cmake</buildtool_depend>
+
+  <depend>ccccc</depend>
+  <depend>bbbbb</depend>
+  <depend>ddddd</depend>
+  <depend>eeeee</depend>
+  <depend>aaaaa</depend>
```

Comment:
What are these depends for?

Reply:
Ah, this is for test_sort_package_xml.
Okay, I'll split this into another PR.

Reply:
#35

Reply:
Removed the test for sort-package-xml from this PR. https://github.com/tier4/pre-commit-hooks-ros/pull/32/commits/e6663a2ba8d6f1d4fefc0defc69acd1bf7017696